src/qibojit/backends/clifford_operations_gpu.py

```diff
@@ -0,0 +1,726 @@
+from functools import cache
+
+import cupy as cp  # pylint: disable=E0401
+import numpy as np
+from scipy import sparse
+
+name = "cupy"
+
+np = cp
+
+GRIDDIM, BLOCKDIM = 1024, 128
+GRIDDIM_2D = (1024, 1024)
+
+
+@cache
+def _get_dim(nqubits):
+    return 2 * nqubits + 1
+
+
+apply_one_qubit_kernel = """
```

Comment:
If you wanna get overly precise with `pylint` suggestions of `python` norms, all these constants defined outside functions should have all characters in uppercase

Reply:
Also, could these constants be moved to the top and be defined together and then all the functions be defined in the end? It's a bit confusing to read the file right now